### ISSUE TITLE
[Suggestion] Unify product names in Android

### ISSUE THREAD

**Issue by MS-PC** [AdguardTeam/ContentBlocker#175]
@ameshkov
You could unify product names in Android. The suggestion is for the website, I do not know if it is in the application. It's just a suggestion, I do not force anyone.

Example:
AdGuard Content Blocker > **AdGuard for Android**
AdGuard for Android > **AdGuard Pro for Android**

Product references:
AdGuard for iOS
AdGuard Pro for iOS

So in Android as in iOS, the "Pro" version will be those that have advantages/features compared to the convecional version that will be limited to compatible browsers.

**Comment by ameshkov** [comment]
Huh, that's a good question actually, we will consider it.

**Comment by lancelot-moon** [comment]
In AG for Android, there are standard, mobile and premium licenses.
If letting AdGuard for Android -> AdGuard Pro for Android, I think users may be confused between pro and premium.

Actually, I don't think AdGuard Content Blocker can be called as AdGuard for Android since it's only a Content Blocker.

Especially, I suggest to push AdGuard for Android (HTTPS filtering is only for browsers).
Let AdGuard Content Blocker keep what it is.

**Comment by superlex** [comment]
Hi :)
I think is more significant AdGuard Content Blocker + AdGuard for Android than AdGuard for iOS + AdGuard Pro for iOS, because AdGuard for iOS is only for Safari mobile and not for all iOS system . So, I'd rather rename AdGuard for iOS -> AdGuard For Safari Mobile.

> Actually, I don't think AdGuard Content Blocker can be called as AdGuard for Android since it's only a Content Blocker.

This.

**Comment by lancelot-moon** [comment]
> AdGuard for iOS -> AdGuard For Safari Mobile

No, free AG for iOS v3 will have the feature of encrypted DNS changer, not only for Safari.

**Comment by superlex** [comment]
@lancelot-moon 
Uhm.. from https://github.com/AdguardTeam/AdguardForiOS I read

> AdGuard for iOS is a Content blocker for Safari browser.

If AG for iOS v3 will have the feature of encrypted DNS changer, maybe AG for iOS v2 should be renamed AdGuard For Safari Mobile or AdGuard Content Blocker (or similar).
However, this is only my opinion :)

**Comment by artemiv4nov** [comment]
So, we will update app names:
Content Blocker -> AdGuard Lite
AdGuard -> AdGuard

**Comment by lancelot-moon** [comment]
Content Blocker is enough to understand.
What does lite mean from the name?
Or please give the subtitle:
`AdGuard Lite - Content Blocker`

**Comment by artemiv4nov** [comment]
We'll see.

**Comment by ameshkov** [comment]
This is not final, but I quite like the idea as people don't fully understand the difference between the content blocker and the full app.

We could extend it even more:

* AdGuard Content Blocker --> AdGuard Lite
* AdGuard for Safari --> AdGuard Lite
* AdGuard Browser Extension --> AdGuard Lite

**Comment by partev** [comment]
I think "AdGuard Content Blocker" should be renamed to "AdGuard Blocker" or just "AdGuard", because the last word is not visible on Android app drawer anyways (see attached screenshot).

All apps have their full name displayed in the drawer except "AdGuard Content Blo..."

Please consider shortening the name so that people understand what they are using.

![Screenshot_20220225-113949_One UI Home](https://user-images.githubusercontent.com/23830955/155755341-9e108adc-9350-4600-89d9-c37a260c8f48.jpg)
